railties/lib/rails/code_statistics.rb

```diff
@@ -4,6 +4,31 @@
 require "active_support/core_ext/enumerable"
 
 class CodeStatistics # :nodoc:
```

Comment:
This class is nodoc, we can't add a method we expect people to use on it and keep it nodoc.

Reply:
Lets remove the `nodoc` and document this class

Reply:
If we are going to document this class should it also be namespaced under `Rails::` like most classes in `railties/lib/rails`?

Reply:
@p8 I think this should be inside the `Rails::` constant, to match the path

Reply:
🙂